src/mdns/mdns_mdnssd.cpp

```diff
@@ -669,28 +670,28 @@ void PublisherMDnsSd::UnsubscribeService(const std::string &aType, const std::st
 
     mSubscribedServices.erase(it);
 
-    otbrLogInfo("unsubscribe service %s.%s (left %zu)", aInstanceName.c_str(), aType.c_str(),
+    otbrLogInfo("Unsubscribe service %s.%s (left %zu)", aInstanceName.c_str(), aType.c_str(),
                 mSubscribedServices.size());
 }
 
 void PublisherMDnsSd::OnServiceResolveFailed(const std::string & aType,
                                              const std::string & aInstanceName,
                                              DNSServiceErrorType aErrorCode)
 {
-    otbrLogWarning("Service %s.%s resolving failed: code=%d", aInstanceName.c_str(), aType.c_str(), aErrorCode);
+    otbrLogWarning("Service %s.%s resolving failed: code=%" PRId32, aInstanceName.c_str(), aType.c_str(), aErrorCode);
```

Comment:
nit: unity log style and make it easy to filter logs using the phrase like `Resolve Service`

```suggestion
    otbrLogWarning("Resolve service %s.%s failed: code=%" PRId32, aInstanceName.c_str(), aType.c_str(), aErrorCode);
```